### DISCUSSION TITLE
Nous L12Z Calibration time

### DISCUSSION THREAD

**Discussion by GregoryMasson** [Koenkk/zigbee2mqtt#17964] [Devices]
Hi !

I have 2 devices Nous L12Z to control 2 blinds.
[https://www.zigbee2mqtt.io/devices/L12Z.html](url)
On the first, when I put calibration to "on" calibration_time started count, at the end it's OK and I can control perfectly this blind.
![image](https://github.com/Koenkk/zigbee2mqtt/assets/50381479/42f4d04a-a3d6-457d-862e-87ae086d7600)

For the second device, the calibration_time is blocked to 10 seconds, when the calibration is end, the blind stop it move before the end of the course
![image](https://github.com/Koenkk/zigbee2mqtt/assets/50381479/3200b77c-9c5d-4a8d-bdf7-b440c2a1aeb2)

I searched in other params, but I'm not an expert and I didn’t find anything for that
Do you have an idea ?

I'm using Zigbee2MQTT installed on Home Assistant OS

Thanks a lot

(sorry for my bad english)

**Comment by raresvintea** [comment]
Any solution? :)

**Reply by GregoryMasson** [reply]
Hi,
sorry for the delay, not the best solution but I create Scripts for all actions
![image](https://github.com/Koenkk/zigbee2mqtt/assets/50381479/85d94378-936e-45b7-bd9e-06bb7a5d3590)

And for Opening and Closing action, I configure a delay for the opening or closing time
![image](https://github.com/Koenkk/zigbee2mqtt/assets/50381479/93369f95-5205-4c16-9041-07af8c8248f5)

**Comment by rogervdh** [comment]
Today I have installed the Nous L12Z module and I have the exact same problem. The callibration time is 10 seconds and can't  be changed in Zigbee2MQTT or Home Assistant. The Zigbee2MQTT logo is shown on the box of the product, so it should be compatible. I will drop a message to Nous. If anyone has a clue, please let us know. Thanks.

**Reply by GregoryMasson** [reply]
Hi rogervdh,
thanks for your answer, you can use scripts to avoid the problem, see my previous response to raresvintea
If you have a problem to configure this, I can help you
